src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
You can take out the declaration of such constants in the .env file . It will be much more convenient and safer.
And don't forget to add .env file in .gitignore so that no one sees your secrets 🙂
Read [here](https://medium.com/how-to-react/using-env-file-in-react-js-b2714235e77e) how to do this.